dns/dnssec.py

```diff
@@ -15,7 +15,8 @@
 
 """Common DNSSEC-related functions and constants."""
 
-import cStringIO
+import six
+from six import BytesIO
```

Comment:
Thanks for the tip. Applied. Bonus point: that highlighted a bunch of inconsistencies on string_types that I missed :)